### DISCUSSION TITLE
How email templates are converted from mjml inside backend/app/email_template to html

### DISCUSSION THREAD

**Discussion by bikash119** [fastapi/full-stack-fastapi-template#1309] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
I do not have any example code.
```

### Description

I would like to understand how the .mjml files inside backend/app/email_templates/src are converted to .html. Is this conversion part of the build process files are converted somewhere outside the build process and pasted into the project.
If it is not handled by build process, does it align with project road map to include this step as part of the build process?

### Operating System

macOS

### Operating System Details

NA

### Python Version

3.12.4

**Comment by alejsdev** [comment, marked as answer]
Hi @bikash119 
Thanks for the feedback. I just realized this was missing in the docs so I've added in #1311, here you can see the process, although it is still very manual. We plan to improve this flow in a future update of this template :rocket:

**Comment by bikash119** [comment]
Thank you @alejsdev
